Makefile

```
@@ -1,16 +1,16 @@
 .PHONY: dev
```

This is the only Makefile which just runs the dev server without any help. It caught me off guard that it did something when just running make.
```suggestion
.PHONY: help
help: 
        @echo "make help              Show this help message"
        @echo "make dev               Run the app in the development server"
        @echo "make shell             Launch a Python shell in the dev environment"
        @echo "make clean             Delete development artefacts (cached files, "
        @echo "                       dependencies, etc)"

.PHONY: dev
```